.gitignore

```diff
@@ -1,3 +1,3 @@
-rest/models/
+rest/models/model_*
+rest/openapi.yaml
```

Comment:
The `rest/openapi.yaml` should be committed and not ignored

Reply:
what's is the reason behind committing it if we get it it from asset-components service?

Reply:
recovered